packages/components/src/FormatFile/FormatFile.mdx

```diff
@@ -61,11 +68,45 @@ following:
 - FormatFile components should take up the full width of the parent container (1
   or 2 files per row).
 
-## Props
+### Display
 
-<Props of={FormatFile} />
+Format files can be displayed as either a file row or a thumbnail.
```

Comment:
I almost think there should be an example of the file row like we have for thumbnail.